public/app/features/alerting/unified/components/notification-policies/Modals.tsx

```diff
@@ -152,32 +156,31 @@ const useEditPolicyModal = (
           )}
         </Modal>
       ),
-    [alertManagerSourceName, error, handleDismiss, handleSave, isDefaultPolicy, loading, route, showModal]
+    [loading, showModal, handleDismiss, error, isDefaultPolicy, route, alertManagerSourceName, handleUpdate, setError]
```

Comment:
Lint rule doesn't mind about `setError` being in dependencies here btw 👀

Reply:
Makes sense right, since `setError` is called in the click handler :)

Reply:
Yup, I meant it as in, it can be removed if it doesn't cause any issues :+1: